frameworks/kafka/tests/test_check_topic_list_on_zk_restart.py

```diff
@@ -103,78 +67,49 @@ def kafka_server(zookeeper_server):
             parse_json=True,
         )[1]["dns"]
 
-        sdk_install.uninstall(
-            config.PACKAGE_NAME, config.SERVICE_NAME
-        )
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
         config.install(
             config.PACKAGE_NAME,
             config.SERVICE_NAME,
             config.DEFAULT_BROKER_COUNT,
-            additional_options={
-                "kafka": {
-                    "kafka_zookeeper_uri": ",".join(
-                        zookeeper_dns
-                    )
-                }
-            },
+            additional_options={"kafka": {"kafka_zookeeper_uri": ",".join(zookeeper_dns)}},
         )
 
         # wait for brokers to finish registering before starting tests
-        test_utils.broker_count_check(
-            config.DEFAULT_BROKER_COUNT,
-            service_name=config.SERVICE_NAME,
-        )
+        test_utils.broker_count_check(config.DEFAULT_BROKER_COUNT, service_name=config.SERVICE_NAME)
 
-        yield {
-            "package_name": config.PACKAGE_NAME,
-            "service": {"name": config.SERVICE_NAME},
-        }
+        yield {"package_name": config.PACKAGE_NAME, "service": {"name": config.SERVICE_NAME}}
     finally:
-        sdk_install.uninstall(
-            config.PACKAGE_NAME, config.SERVICE_NAME
-        )
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
 
 
 def topic_create(kafka_server: dict):
-    return test_utils.create_topic(
-        config.EPHEMERAL_TOPIC_NAME,
-        kafka_server["service"]["name"],
-    )
+    return test_utils.create_topic(config.EPHEMERAL_TOPIC_NAME, kafka_server["service"]["name"])
 
 
 def fetch_topic(kafka_server: dict):
     _, topic_list, _ = sdk_cmd.svc_cli(
-        config.PACKAGE_NAME,
-        kafka_server["service"]["name"],
-        "topic list",
-        parse_json=True,
+        config.PACKAGE_NAME, kafka_server["service"]["name"], "topic list", parse_json=True
     )
     return topic_list
 
 
-def restart_zookeeper_node(id: int):
+def restart_zookeeper_node(id: int, kafka_server: dict):
```

Comment:
`kafka_server` argument here is unused, did you mean to do something with it here?

Reply:
While restarting zookeeper pods, if it takes a while, the kafka basiclly uninstalls. To keep it alive, I kept the engagement of kafka service `yeild` active.  This prevents the kafka uninstalls between the tests.

Reply:
@kaiwalyajoshi , merging this if your comment has been addressed?